### PR TITLE
feat: Add arguments in EFS file system

### PR DESCRIPTION
**Pull request by titanlien**
## Description

1. Be able to import EFS file system created by the other terraform module

## Breaking Changes

No

## How Has This Been Tested?
- [ ] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -446,7 +446,7 @@ resource "aws_route53_record" "atlantis_aaaa" {
 resource "aws_efs_file_system" "this" {
   count = var.enable_ephemeral_storage ? 0 : 1
 
-  creation_token = var.name
+  creation_token = var.efs_file_system_token != "" ? var.efs_file_system_token : var.name
```
Comment (bryantbiggs) [inline main.tf:449]:
```suggestion
  creation_token = coalesce(var.efs_file_system_token, var.name)
```

**Comment by bryantbiggs** [comment]
@titanlien it looks like I can't update your branch - could you please take a look at the change above and update with `master`, then we should be able to merge

**Comment by titanlien** [comment]
Hi @cytopia I believe you can fetch this upstream and update our PR again.

**Comment by snovikov** [comment]
@bryantbiggs updated

**Review by bryantbiggs — APPROVED** [review]
thank you

**Comment by antonbabenko** [comment]
This PR is included in version 3.20.0 :tada:

**Pull request merged**